src/pretix/control/forms/vouchers.py

```diff
@@ -234,8 +234,8 @@ def clean(self):
         )
         if check_quota:
             Voucher.clean_quota_check(
-                data, cnt, self.initial_instance_data, self.instance.event,
-                self.instance.quota, self.instance.item, self.instance.variation
+                data, cnt, self.initial_instance_data,
+                self.instance.event, self.instance.quota, self.instance.item, self.instance.variation
```

Comment:
Basically undo the linebreak introduced by the `if` that was originally in the PR
```suggestion
                data, cnt, self.initial_instance_data, self.instance.event,
                self.instance.quota, self.instance.item, self.instance.variation
```